### PR TITLE
Make Custom column name feature available in:

### PR DESCRIPTION
**Pull request by Shiv-Chata**
- DM component
- Dashboards
- Data Alerts

### CODE REVIEW

File: src/components/AddColumnBtn/CustomColumnModal.js [hunk 1]
```diff
@@ -501,6 +509,26 @@ export default class CustomColumnModal extends React.Component {
     return !!this.state.selectedFnType && !!this.state.selectedFnColumn
   }
 
+  checkColumnName = (nameVal) => {
+    const columnNameExists = this.props?.columns?.find((col) => col?.display_name === nameVal)
+    if (columnNameExists || !nameVal) {
```
Comment (eleon-chata-ai) [inline src/components/AddColumnBtn/CustomColumnModal.js:514]: can remove if to return !(columnNameExists || !nameVal)

File: src/components/Input/Input.scss [hunk 2]
```diff
@@ -36,6 +37,7 @@
       }
 
       &:hover {
+
```
Comment (eleon-chata-ai) [inline src/components/Input/Input.scss:40]: extra white space

File: src/components/Input/Input.scss [hunk 3]
```diff
@@ -276,4 +284,4 @@
     margin: 0 3px;
     opacity: 0.6;
   }
-}
+}
```
Comment (eleon-chata-ai) [inline src/components/Input/Input.scss:287]: don't think the extra space at the bottom of the page is needed?

**Review by eleon-chata-ai — CHANGES_REQUESTED** [review]

File: src/components/Input/Input.scss (hunk 3 above)
Comment (Shiv-Chata) [reply src/components/Input/Input.scss:287]: This is still at the end of the file, with no extra space. It is just that it has moved down the file from line 279 to 287

**Review by eleon-chata-ai — CHANGES_REQUESTED** [review]

File: src/components/Input/Input.scss (hunk 3 above)
Comment (Shiv-Chata) [reply src/components/Input/Input.scss]: I think that fixed it

**Review by eleon-chata-ai — APPROVED** [review]

**Pull request merged**